lib/template.html

```diff
@@ -51,7 +51,11 @@
 
 {% if site.twitter %}
   {% if seo_tag.image %}
-    <meta name="twitter:card" content="summary_large_image" />
+    {% if page.twitter_image_small or post.twitter_image_small %}
+      <meta name="twitter:card" content="summary" />
+    {% else %}
+      <meta name="twitter:card" content="summary_large_image" />
+    {% endif %}
```

Comment:
There are already other types of cards (`app` and `player` for now): https://dev.twitter.com/cards/types.